app/views/CannedResponseDetail.js

```diff
@@ -0,0 +1,171 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { StyleSheet, Text, View, ScrollView } from 'react-native';
+import { useSelector } from 'react-redux';
+
+import I18n from '../i18n';
+import SafeAreaView from '../containers/SafeAreaView';
+import StatusBar from '../containers/StatusBar';
+import Button from '../containers/Button';
+import { useTheme } from '../theme';
+import RocketChat from '../lib/rocketchat';
+import Navigation from '../lib/Navigation';
+import { goRoom } from '../utils/goRoom';
+import { themes } from '../constants/colors';
+import Markdown from '../containers/markdown';
+import sharedStyles from './Styles';
+
+const styles = StyleSheet.create({
+	scroll: {
+		flex: 1
+	},
+	container: {
+		flex: 1,
+		marginTop: 12,
+		marginHorizontal: 15
+	},
+	cannedText: {
+		marginTop: 8,
+		marginBottom: 16,
+		fontSize: 14,
+		paddingTop: 0,
+		paddingBottom: 0,
+		...sharedStyles.textRegular
+	},
+	cannedTagWrap: {
+		borderRadius: 4,
+		marginRight: 4,
+		marginTop: 8,
+		height: 16
+	},
+	cannedTagContainer: {
+		flexDirection: 'row',
+		flexWrap: 'wrap'
+	},
+	cannedTag: {
+		fontSize: 12,
+		paddingTop: 0,
+		paddingBottom: 0,
+		paddingHorizontal: 4,
+		...sharedStyles.textRegular
+	},
+	button: {
+		margin: 24,
+		marginBottom: 24
+	},
+	item: {
+		paddingVertical: 10,
+		justifyContent: 'center'
+	},
+	itemLabel: {
+		marginBottom: 10,
+		fontSize: 14,
+		...sharedStyles.textMedium
+	},
+	itemContent: {
+		fontSize: 14,
+		...sharedStyles.textRegular
+	}
+});
+
+const Item = ({ label, content, theme, testID }) =>
```

Comment:
this `const Item` is duplicated, we have the same code at this file: app/views/RoomInfoView/Item.js

@diegolmello Maybe, can we create a new component called Item, on our containers. What do you think ?

Reply:
@AlexAlexandre Agreed, but we can do it on a separate PR.
We've been working on the design system, so it's going to be refactored anyway.